src/Blockcore.Indexer.Core/Controllers/QueryController.cs

```diff
@@ -39,6 +40,18 @@ public IActionResult GetAddress([MinLength(4)][MaxLength(100)] string address)
          return Ok(storage.AddressBalance(address));
       }
 
+      /// <summary>
+      /// Get the balance of multiple addresses.
+      /// </summary>
+      /// <param name="addresses"></param>
+      /// <returns></returns>
+      [HttpPost]
```

Comment:
I notice this is an http post, I guess we have to do that becuase we can't specify multiple addresses in the query string and get doesn't read from the body right?

It seems in this post that putting a body ion a get is actually fine now
https://stackoverflow.com/questions/978061/http-get-with-request-body

Thoughts?